components/atoms/AuthorCard.tsx

```diff
@@ -11,12 +11,9 @@ interface Props {
 
 const ContentCard: React.FunctionComponent<Props> = props => (
     <div className="card mb-4">
-        {props.image ?
-          <img src={props.image} alt="Author Avatar" className="card-img-top rounded-circle" /> :
-          <img src="/authors/unisex-avatar.svg" alt="Author Avatar" className="card-img-top rounded-circle p-4" />}
         <div className="card-body">
             <h5 className="card-title text-center">
-                <Link href='#'>
+                <Link href={`/authors/${props.id}`}>
```

Comment:
So I personally think that, because we are essentially using slugs as ids right now, we should defer to the convention of calling slugs slugs. In `FrontMatter`, instead of `author`, we could have `authorSlug`, and in `AuthorCollection` we could have `slug`. This will make it simpler, I think, when we actually start using meaningful ids. It also is easier to comprehend, since the meaning will be straightforward (instead of `id` always actually meaning `slug`).

Reply:
Great feedback, let us use slug instead